lib/new_relic/agent/transaction.rb

```diff
@@ -623,7 +602,7 @@ def calculate_gc_time
       def calculate_transport_duration distributed_trace_payload
         return unless distributed_trace_payload
 
-        duration = (start_time.to_f * 1000 - distributed_trace_payload.timestamp) / 1000
+        duration = start_time - distributed_trace_payload.timestamp
```

Comment:
It seems like the calculation is slightly different now here? did something change with `distributed_trace_payload.timestamp` that makes the updated calculation the same as before?

Reply:
I think we just missed this the first time. Per comment, `distributed_trace_payload.timestamp` is in milliseconds but the duration is expected to be in seconds. Will update this.

```suggestion
        duration = start_time - distributed_trace_payload.timestamp/1000
```